pkg/activator/handler/concurrency_reporter_test.go

```diff
@@ -480,20 +481,23 @@ func TestMetricsReported(t *testing.T) {
 	<-cr.statCh
 	<-cr.statCh
 
+	wantResource := &resource.Resource{
+		Type: "knative_revision",
+		Labels: map[string]string{
+			metricskey.LabelRevisionName:      rev1.Name,
+			metricskey.LabelNamespaceName:     rev1.Namespace,
+			metricskey.LabelServiceName:       "service-" + rev1.Name,
+			metricskey.LabelConfigurationName: "config-" + rev1.Name,
+		},
+	}
 	wantTags := map[string]string{
-		metricskey.LabelRevisionName:      rev1.Name,
-		metricskey.LabelNamespaceName:     rev1.Namespace,
-		metricskey.LabelServiceName:       "service-" + rev1.Name,
-		metricskey.LabelConfigurationName: "config-" + rev1.Name,
-		metricskey.PodName:                "the-best-activator",
-		metricskey.ContainerName:          "activator",
+		metricskey.PodName:       "the-best-activator",
+		metricskey.ContainerName: "activator",
 	}
-	metricstest.CheckLastValueData(t, "request_concurrency", wantTags, 3)
-
+	metricstest.AssertMetric(t, metricstest.FloatMetric("request_concurrency", 3, wantTags).WithResource(wantResource))
```

Comment:
It may be easier to store the metric and then increment it for line 500, and you won't need to repeat the metric names and tags.

Reply:
Done.